index.html

```diff
@@ -0,0 +1,324 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars">
+        <i class="fas fa-bars"></i>
+      </button>
+    </header>
+    <main>
+      <div class="intro">
+        <div class="intro-icon">
+          <img src="./media/svg/icon.svg" alt="Icon for the side" />
+        </div>
+        <h2 class="intro-greet">
+          Hey There.
+          <br />
+          I'm Aziz
+        </h2>
+        <h2 class="intro-me">I’am a Software Developer</h2>
+        <p class="intro-text">
+          I can help you build a product, feature or website <br />
+          Look through some of my work and experience! <br />
+          If you like what you see and have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
+        <div class="social">
+          <a
+            class="social-link"
+            href="https://github.com/SpaYco"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            <img
+              class="social-icon"
+              src="./media/svg/github.svg"
+              alt="GitHub Icon"
+            />
+          </a>
+          <a
+            class="social-link"
+            href="https://www.linkedin.com/in/spayco"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            <img
+              class="social-icon"
+              src="./media/svg/linkedin.svg"
+              alt="LinkedIn Icon"
+            />
+          </a>
+          <a
+            class="social-link"
+            href="https://angel.co/spayco"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            <img
+              class="social-icon"
+              src="./media/svg/angel.svg"
+              alt="AngelList Icon"
+            />
+          </a>
+          <a
+            class="social-link"
+            href="https://twitter.com/iSpaYco"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            <img
+              class="social-icon"
+              src="./media/svg/twitter.svg"
+              alt="Twitter Icon"
+            />
+          </a>
+          <a
+            class="social-link"
+            href="https://spayco.medium.com/"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            <img
+              class="social-icon"
+              src="./media/svg/medium.svg"
+              alt="Medium Icon"
+            />
+          </a>
+        </div>
```

Comment:
Hey, @sja-thedude Great job for creating this social media section However It's supposed to be a list. You can improve this by replacing `<div>` tag and `<a>` tag with `<ul>` tag and `<li>` because it helps screen readers easily understand and in reading. 

I'll suggest you when you see something like a list while building a website it's a good practice to use `<ul>` and `<li>` tags in combination. It's highly recommended for Semantic and SEO perspective.